java/src/main/java/io/cucumber/java/Step.java

```diff
@@ -0,0 +1,68 @@
+package io.cucumber.java;
+
+import io.cucumber.core.backend.TestCaseState;
+import io.cucumber.plugin.event.HookTestStep;
+import io.cucumber.plugin.event.Location;
+import io.cucumber.plugin.event.PickleStepTestStep;
+import io.cucumber.plugin.event.StepArgument;
+import io.cucumber.plugin.event.TestStep;
```

Comment:
The `java` module and the `plugin` module evolve at different speeds. Users of the java module should only ever have to import classes from `io.cucumber.java`. At a glance `Location` and `StepArgument` are currently exposed.

Reply:
Note: Currently this is not enforced but will/may be in the future once we support `module-info.java` in all modules.